File: Iniciante/2061/2061.c
#include <stdio.h>
#include <string.h>

int main()
{
   int N, i, M;
   char str[20];
   
   scanf("%d %d", &N, &M);
   
   for (i = 0; i < M; i++)
   {
      scanf("%s", str);
      
      if(strcmp(str, "fechou")==0){
         N++;
      }
      else if(strcmp(str, "clicou")==0){
         N--;
      }
    
   }
   printf("%d\n", N);
   return 0;
}

File: Python/Iniciante/2061.py
# Problema 2061 - Beecrowd - Iniciante - Nível 1

# Entrada de dados, primeiro o número de abas já abertas e o segundo o valor de ações feitas por Péricles
n_abas,n_acoes = map(int,input().split(" "))

# Recebendo quais ações foram feitas
for i in range(n_acoes):
    acao = input()
    if(acao=="fechou"):
        n_abas +=1
    elif(acao=="clicou"):
        n_abas -=1

# Apresentando o número de abas final
print(n_abas)